traffic_stats/influxdb_tools/create/create_ts_databases.go

```diff
@@ -20,45 +20,81 @@ under the License.
 package main
 
 import (
-	"flag"
 	"fmt"
-	"os"
 
 	influx "github.com/influxdata/influxdb/client/v2"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
 )
 
-func main() {
+const (
+	cache           = "cache_stats"
+	deliveryService = "deliveryservice_stats"
+	daily           = "daily_stats"
+)
+
+func createFlags() []cli.Flag {
+	return []cli.Flag{
```

Comment:
That's definitely a premature optimization which I'd be happy to swap for a named var. Since there were no tests to start with, I had to A/B test most of the changes I was making. My "long" term goal was to abstract out something like a 
```go
type DBConfig struct {
    url string
    replication int
    user string
    password string
} 

func flagName(prefix, name string) string {
   if prefix == "" {
      return name
   }
   return fmt.Sprintf("%s-$s", prefix, name)
}

func envName(prefix, name string) string {
   if prefix == "" {
      return strings.ToUpper(name)
   }
   return strings.ToUpper(fmt.Sprintf("%s_$s", prefix, name))
}

func (c *DBConfig) Flags(prefix string) []cli.Flag {
	return []cli.Flag{
		cli.StringFlag{
			Name:  flagName(prefix, "url"),
                        EnvVar: envName(prefix, "url"),
			Usage: "The influxdb url and port",
			Value: "http://localhost:8086",
                        Destination: &c.url,
		},
		...
	}
}
```

Which would allow the start of some abstraction between the two commands, however, I realized part way that my changes were already larger than recommended, and that the course I was taking at the time (one executable with two commands) would actually break the build script as it stands.